features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -95,6 +98,77 @@ export default function ListBlacklistThrottlingPolicies() {
         }),
     };
 
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
+        let incrementId = 0;
+        return new Promise(((resolve, reject) => {
+            restApi.blacklistPoliciesGet().then((result) => {
+                policyList = result.body.list;
+                const blacklistPolicies = policyList.map((obj) => {
+                    let array = [];
+                    incrementId++;
+                    if (obj.conditionValue === Object(obj.conditionValue)) {
```

Comment:
Why we use [`Object`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) constructor here ?

This will convert `null` or `undefined` values to objects ?

```
The Object constructor creates an object wrapper for the given value.

If the value is null or undefined, it will create and return an empty object.
Otherwise, it will return an object of a Type that corresponds to the given value.
If the value is an object already, it will return the value.
```

Reply:
This helps us identify which conditional values need to be converted to an array.
<img width="749" alt="Screen Shot 2020-05-27 at 6 05 20 PM" src="https://user-images.githubusercontent.com/19324135/83019584-d4679580-a044-11ea-9e4b-573661a6cd87.png">